src/main/java/com/srans/nestserver/repository/NotificationRepository.java

```diff
@@ -15,7 +15,7 @@
 	public Long[] getNotificationId();
 
 	//get all new notification
-	@Query(value = "select n.id, n.message from notification n  inner JOIN notification_user nu on (n.id=nu.notification_id) where nu.view_status='N' and nu.user_id=?1", nativeQuery = true)
+	@Query(value = "select n.id, n.message, n.tenant_id from notification n  inner JOIN notification_user nu on (n.id=nu.notification_id) where nu.view_status='N' and nu.user_id=?1", nativeQuery = true)
 	public List<Object[]> getAllNotification(Long userId);
```

Comment:
Why can't u use jpa query and reduce the tediousnees of handling List objects

Reply:
Ok sir, I have changed...